templates/Default/engine/Default/game_join.php

```diff
@@ -146,25 +145,8 @@
 				</table>
 			</td>
 		</tr>
-
-		<tr>
-			<td>
-				<table class="center">
-					<tr>
-						<td colspan="3">Trading</td>
-					</tr>
-					<tr>
-						<td>Combat<br />Strength</td>
-						<td>
-							<img width="440" height="440" id="graphframe" src="images/race/graph/graph<?php echo $SelectedRaceID; ?>.gif" alt="Race overview" />
-						</td>
-						<td>Hunting</td>
-					</tr>
-					<tr>
-						<td colspan="3">Utility</td>
-					</tr>
-				</table>
-			</td>
-		</tr>
 	</table>
+	<div class="center">
+		<div style="width: 600px; height: 600px;background: none; margin: auto;" id="graphframe"></div>
```

Comment:
The `margin` change I recommended above allows us to remove all the `style` elements here.
```suggestion
		<div id="graphframe"></div>
```